src/style.css

```
@@ -0,0 +1,72 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+.search {
+  width: 100%;
+  box-sizing: border-box;
+  height: 70px;
+
+  padding-left: 62px;
+  padding-bottom: 2px;
+
+  background: #fff;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+  font-family: "Avenir", Helvetica, sans-serif;
+}
+
+.search-2 {
+  height: 42px;
+  padding-left: 33px;
+
+  font-size: 14px;
+  line-height: 19px;
+  box-shadow: none;
+}
+
+.search_form:first-child {
+  margin: 20px 0 -3px;
+}
```

No need for this `-3` margin
```suggestion
.search_form:first-child {
  margin: 20px 0;
}
```